src/memgraph.cpp

```diff
@@ -98,6 +98,10 @@
 #include "audit/log.hpp"
 #endif
 
+const char *const MG_USER = "MG_USER";
+const char *const MG_PASSWORD = "MG_PASSWORD";
+const char *const MG_PASSFILE = "MG_PASSFILE";
```

Comment:
```suggestion
constexpr char *const kMgUser = "MG_USER";
constexpr char *const kMgPassword = "MG_PASSWORD";
constexpr char *const kMgPassfile = "MG_PASSFILE";
```

Reply:
I had to change constexpr to const because of clang-tidy.

Reply:
What is the error from clang-tidy?

Reply:
no no it was my mistake and maybe I did not communicate it very well on our private chat. The error was about the `pointer itself` being global and `non const`. I think it's safe to use `constexpr` here and clang-tidy won't complain as long as the `pointer` itself is `const` (not only a pointer to const but a const pointer to a const object)

Reply:
Ah yes, something like this `constexpr const char *kMgUser`

Reply:
seems like this is resolved, resolving the conversation